### PR TITLE
adding about me section

### PR DESCRIPTION
**Pull request by boo-shehab**
I added a design for the about me section for mobile
It's including my project that's published, information about me, and my skills.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,11 +34,11 @@
     </div>
```
Comment (fobadara) [inline index.html:34]: [OPTIONAL] This is a requirement for the previous project so I had to make it optional. The background image for the template surrounds the text in the first page but yours moves towards the left.

| Your Design |
|-------------|
|image|

File: index.html [hunk 2]
```diff
@@ -51,5 +51,129 @@ <h1>I’m Alexandra Glad to see you!</h1>
...
+                            <li>2015</li>
+                        </ul>
+                    </div>
+                    <p class="info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="tages">
+                        <span class="tage">html</span>
+                        <span class="tage">css</span>
+                        <span class="tage">javaScript</span>
+                    </div>
+                    <button class="normal-button">See Project</button>
```
Comment (fobadara) [inline index.html:114]: - [ ] [REQUIRED] I noticed that all your see project buttons are not following the template style during hover and active state. Kindly fix that.

| Template Buttons |
|------------------|
|image|


- [ ] [REQUIRED] Kindly ensure that all your see project buttons are well aligned to the left just like the text above them

|Your Button|
|-----------|
|image|

File: index.html [hunk 3]
```diff
@@ -51,5 +51,129 @@ <h1>I’m Alexandra Glad to see you!</h1>
...
```
Comment (fobadara) [inline index.html:134]: - [ ] [REQUIRED] I noticed that all your see project buttons are not following the template style during hover and active state. Kindly fix that.

| Template Buttons |
|------------------|
|image|




- [ ] [REQUIRED] Kindly ensure that all your see project buttons are well aligned to the left just like the text above them

|Your Button|
|-----------|
|image|

File: index.html [hunk 4]
```diff
@@ -51,5 +51,129 @@ <h1>I’m Alexandra Glad to see you!</h1>
...
+                <div class="card-work">
+                    <img src="img/Snapshoot Portfolio (1).png" alt="">
+                    <h2 class="title">Multi-Post Stories</h2>
+                    <div class="client">
+                        <span>CANOPY</span>
+                        <ul>
+                            <li>Back End Dev</li>
+                            <li>2015</li>
+                        </ul>
+                    </div>
```
Comment (fobadara) [inline index.html]: - [ ] [REQUIRED] I noticed that the more height given to your design the more distanced the cards for the work section are. Kindly ensure that the distance between the landing page and the first card match the template and also the distance between each card.

|Your Design|
|---------------|
|boo-shehab1|

File: index.html [hunk 5]
```diff
@@ -51,5 +51,129 @@ <h1>I’m Alexandra Glad to see you!</h1>
...
+            <article class="card">
+                <div class="card-work">
+                    <img src="img/Snapshoot Portfolio.png" alt="">
+                    <h2 class="title">Tonic</h2>
+                    <div class="client">
+                        <span>CANOPY</span>
+                        <ul>
+                            <li>Back End Dev</li>
+                            <li>2015</li>
+                        </ul>
```
Comment (fobadara) [inline index.html:66]: - [ ] [REQUIRED] Kindly take note of variations like the one present in the image below when screen size increase up from 375px until we reach desktop size. Please make sure it still meets the portfolio template.

|Your Design|
|-------------|
|image|

File: index.html [hunk 6]
```diff
@@ -51,5 +51,129 @@ <h1>I’m Alexandra Glad to see you!</h1>
...
+                    </div>
+                    <p class="info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="tages">
+                        <span class="tage">html</span>
+                        <span class="tage">css</span>
+                        <span class="tage">javaScript</span>
+                    </div>
+                    <button class="normal-button">See Project</button>
```
Comment (fobadara) [inline index.html]: - [ ] [REQUIRED] Kindly ensure this also follows the template button during all the states.
- [ ] [REQUIRED] Kindly ensure that all your see project buttons are well aligned to the left just like the text above them

|Your Button|
|-----------|
|image|

File: index.html [hunk 7]
```diff
@@ -51,5 +51,129 @@ <h1>I’m Alexandra Glad to see you!</h1>
...
+                <p>LET’S CONNECT</p>
+                <ul>
+                    <li><a href="#"><img src="img/twitter.jpg" alt=""></a></li>
+                    <li><a href="#">in</a></li>
+                    <li><a href="#"><img src="img/medium.jpg" alt=""></a></li>
+                    <li><a href="#"><img src="img/github.jpg" alt=""></a></li>
+                    <li><a href="#"><img src="img/angellist.jpg" alt=""></a></li>
+                </ul>
+            </div>
+            <button class="normal-button">Get my resume</button>
```
Comment (fobadara) [inline index.html:157]: - [ ] [REQUIRED] Kindly ensure your button follows the template in all states.

| Your Design |
|--------------|
|image|

- [ ] [REQUIRED] Kindly ensure the button. is well aligned to the left

**Review by fobadara — CHANGES_REQUESTED** [review]
# REQUIRES CHANGES ♻️ 
Hi @boo-shehab, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @boo-shehab, 

## Status: APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 
- Great job with your design and functionalities 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**